docs/FAQ.md

```diff
@@ -52,7 +57,22 @@ Here is a JSON result that we receive from function above:
     6
 ]
 ```
+
 When it is processed on Qt side, it means that function 2 from Module 27 should be invoked with passed params. (react-native knows module ids from config).
 
 ### Where can I read some intro on React Native internals?
+
 https://www.logicroom.co/react-native-architecture-explained/, https://facebook.github.io/react-native/docs/communication-ios.html.
+
+### How do I control Qt logging?
+
+The app leverages [QLoggingCategory](http://doc.qt.io/qt-5/qloggingcategory.html#configuring-categories) to allow filtering of categories. If the app sets the default categories using [QLoggingCategory::setFilterRules](http://doc.qt.io/qt-5/qloggingcategory.html#setFilterRules), then the categories can only be overriden using the `QT_LOGGING_CONF` and `QT_LOGGING_RULES` environment variables.
+
+Order of evaluation:
+
+1. `[QLibraryInfo::DataPath]/qtlogging.ini`
+1. `QtProject/qtlogging.ini`
+1. `setFilterRules()`
+1. `QT_LOGGING_CONF`
+1. `QT_LOGGING_RULES`
+
```

Comment:
@PombeirP, thanks for doc.
I think we can add an example related to react-native-desktop to make it more clear:
```
QT_LOGGING_RULES="*VIEWMANAGER=false;NETWORKING=true"
```

Btw, is there a way to somehow list all categories used within the `react-native-desktop`? That could be helpful for users. If no, probably we need to list them in documentation.

Reply:
Agree with the example. I don't think there's any way to list the categories, as that is spread all around the code.